### PR TITLE
feat(control-flow): migrate to the new control-flow via schematic command

### PR DESCRIPTION
**Pull request by bougwal**
Hey @falberthen  I hope you are well! 

So there is a lot going on on Angular these days and EcomDDD is behind with a lot of features... 
I got snowed under with a lot of professional commitments the past few weeks. 

I just ran the new Control Flow Syntax migration schematic command in the FE project to keep up with the upcoming changes. This is an automatic transformation by Angular utilities
____
I am using Colima to run the backend services - For some reason it gets hung and never ends - I assume it's my own env issue. I'll investigate it. 

______
Thus, asking you to have a quick look before merging to check these syntactic sugar changes are fine, as I did not run it locally!

### CODE REVIEW

**Comment by falberthen** [comment]
> Hey @falberthen I hope you are well!
> 
> So there is a lot going on on Angular these days and EcomDDD is behind with a lot of features... I got snowed under with a lot of professional commitments the past few weeks.
> 
> I just ran the new Control Flow Syntax migration schematic command in the FE project to keep up with the upcoming changes. This is an automatic transformation by Angular utilities
> 
> I am using Colima to run the backend services - For some reason it gets hung and never ends - I assume it's my own env issue. I'll investigate it.
> 
> Thus, asking you to have a quick look before merging to check these syntactic sugar changes are fine, as I did not run it locally!

Hey @bougwal, tks for submitting it. I'll test it this weekend and give you feedback. Regarding backend services, how you used to run then before? The way I recommend first running the docker compose.  We keep in touch through dm if you need any help.

File: src/EcommerceDDD.Spa/src/app/shared/components/nav-menu/nav-menu.component.html [hunk 1]
```diff
@@ -4,88 +4,91 @@
...
+        <div class="container justify-content-center">
+          <a class="navbar-brand" href="#"></a>
+          <button
+            class="navbar-toggler"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbar"
+            >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          @if (isLoggedIn) {
```
Comment (falberthen) [inline src/EcommerceDDD.Spa/src/app/shared/components/nav-menu/nav-menu.component.html]: We can safely remove this condition check.

**Review by falberthen — APPROVED** [review]

File: src/EcommerceDDD.Spa/src/app/shared/components/nav-menu/nav-menu.component.html (hunk 1 above)
Comment (bougwal) [reply src/EcommerceDDD.Spa/src/app/shared/components/nav-menu/nav-menu.component.html]: Good catch ! Looks redundant
_______ 
It seems the comments was pending for 2 weeks I did not see it 🧐 - Sorry for the delay Felipe

Comment (bougwal) [reply src/EcommerceDDD.Spa/src/app/shared/components/nav-menu/nav-menu.component.html]: Good catch ! Looks redundant
_______ 
It seems the comment was pending for 2 weeks I did not see it 🧐 - Sorry for the delay Felipe

**Pull request merged**